aws_adfs/login.py

```diff
@@ -1,5 +1,6 @@
 import configparser
-
+import requests
```

Comment:
requests and json imports are unnecessary, I think. I can't see any change in this file that requires them.